app/models/template.rb

```diff
@@ -31,8 +31,9 @@ def self.template_includes
     []
   end
 
-  def preview_host_collection
-    Host.authorized(:view_hosts).order(:name).limit(100)
```

Comment:
I don't think plugins need to override this (at least not REX), the previewing should work the same for each template type. I hope I'm not missing something.